packages/common/src/index.ts

```diff
@@ -1,8 +1,8 @@
 export * from "./cursorlessCommandIds";
 export * from "./testUtil/extractTargetedMarks";
 export { default as FakeIDE } from "./ide/fake/FakeIDE";
-export { Message } from "./ide/spy/SpyMessages";
-export { SpyIDERecordedValues } from "./ide/spy/SpyIDE";
+export type { Message } from "./ide/spy/SpyMessages";
+export type { SpyIDERecordedValues } from "./ide/spy/SpyIDE";
```

Comment:
Are these leftover from #1832 or are they required for this PR?

Reply:
See the build logs for the first commit.